.github/workflows/documentation.yaml

```diff
@@ -17,7 +17,7 @@ concurrency:
 jobs:
   check_changes:
     name: Deduce required tests from code changes
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
```

Comment:
should we go to 22.04 directly ? :thinking:

Reply:
`ubuntu-latest` is currently still pointing to `ubuntu-20.04`, so I think 20.04 is fine for now.